development.md

```diff
@@ -3,21 +3,32 @@ layout: page
 title:  "Development"
 ---
 
-When a developer starts working on a new feature, right after creating the empty branch he opens in GitHub a [pull request](https://help.github.com/articles/using-pull-requests) to merge the feature branch into **master**. This is helpful as everyone can see what the other members of the team are working on by simply checking all the open pull requests.
+When a team member is ready to start working on a new issue, he accesses the `Development` Project and looks, in the 'Backlog' column, for Issues assigned to him.
 
-At every push to GitHub the feature branch goes through [CI](https://inforlife.github.io/process/ci.html) to ensure the new code didn't break anything.
+If he finds one, he moves it to the `In progress` column.
 
-When completed, the feature must be reviewed before it can be merged into **master**. So, the developer assigns the pull request to a  *reviewer* (another member of the team).
-As soon as possible, the reviewer looks through the pull request, adds comments where necessary, and, according to the outcome of the review, approves it or requests changes.
+If no Issues are currently assigned to the team member, he picks up, from the 'Backlog' column, the one above all others (the currently most relevant), assigns it to him and, moves it to the `In progress` column.
 
-Once the review is completed, developer and reviewer meet (physically or remotely) and complete the review by going together through the comments.
+At this point, the team member creates locally an 'issue' branch named like the issue title and implements one after the other all the *Conditions of satisfaction* listed in the Issue using [Test-Driven Development](https://inforlife.github.io/process/test-driven-development.html).
 
-If the pull request contains any code that requires changes, it is updated according to the review before the branch is merged into **master** (and the issue marked as *reviewed*). Otherwise, the merge can be done as soon as the review is completed by the branch owner.
+The new code is pushed regularly to GitHub (at least one per day).
 
-We believe that code review is the single biggest thing we can do to improve the quality of the code we produce and, as very positive side effect, it also improves developers' skills.
+After the first push, the team member opens in GitHub a [Pull request](https://help.github.com/articles/using-pull-requests) to merge the issue branch into `master`.
```

Comment:
Detail that the pull request must reference the issue in the description to ensure full traceability.